gaming-website/docker-compose.yaml

```diff
@@ -0,0 +1,10 @@
+version: "3"
+services:
+    app:
+        build: .
+        ports:
+            - "3000:3000"
+        environment:
```

Comment:
Have you tried not setting these at all? What happens?

Reply:
Well they are not needed calculating also the fact that the vite.config is set to that port to be the default one